wrapper.html

```diff
@@ -101,7 +100,7 @@
 
 {% block meta_additions %}{% endblock %}
 
-	<link rel='stylesheet' id='baseline-css' href='https://dbqvwi2zcv14h.cloudfront.net/ak/ak-v3.css?v=2018091408552' type='text/css' media='screen' />
+	<link rel='stylesheet' id='baseline-css' href='https://dbqvwi2zcv14h.cloudfront.net/ak-dev/ak-v3.css' type='text/css' media='screen' />

```

Comment:
Hey @mrkimani-ndegwa! Just flagging that because we were testing new features for GT, a different folder, `ak-dev`, was created to store these new assets. The original file in `develop` references the `ak` subfolder in s3.350.org bucket. Do we want to keep this change or merge the files in `ak-dev` into `ak` and stick to using the `ak` subfolder?